### PR TITLE
Add tabs scaffold

### PR DESCRIPTION
**Pull request by IagoLast**
#43 Tabs Element
---

This PR creates a new Tabs Element.

- [x] Add tabs element
- [x] Use new tabs element in every layout example
- [x] Add documentation
- [x] Wait for design for approval ✔️ 
- [x] CR
- [x] Add ui-test

### CODE REVIEW

**Comment by piensaenpixel** [comment]
Hi guys some things, thanks!

- [ ] The active state in XL Tabs should be 4px instead of 2
image

- [ ] Padding bottom should be 8px instead of 12px
image

- [ ] Both the active and hover states should be over the border bottom
image

- [ ] I think we don't need the flex-shrink and the text-align
image

One question, why aren't we using a link inside the <li>? I think we should support it. Now, if we add a link, it breaks the styles

image

**Comment by IagoLast** [comment]
Regarding the padding, I think the tabs might be too small to be easily clickable affecting our UX

screen shot 2018-07-31 at 11 27 16


what do you think? @piensaenpixel

**Comment by IagoLast** [comment]
>  I think we don't need the flex-shrink and the ...

 flex-shrink is needed to give horizontal scrolling

**Comment by piensaenpixel** [comment]
@IagoLast regarding the padding i think if we add the the padding to the link i think it can work. And also we are testing in some solutions and it works.

**Comment by piensaenpixel** [comment]
+1 to @rubenmoya comments

File: examples/layouts/generic.html [hunk 1]
```diff
@@ -65,13 +68,13 @@
...
   </header>
 
-  <nav class="as-toolbar-tabs">
-    <span onclick="showMap(event)" class="as-toolbar-tabs__item as-toolbar-tabs__item--active">MAP</span>
-    <span onclick="showLeft(event)" class="as-toolbar-tabs__item">LEFT</span>
-    <span onclick="showRight(event)" class="as-toolbar-tabs__item">RIGHT</span>
-    <span onclick="showLegends(event)" class="as-toolbar-tabs__item">LEGENDS</span>
-    <span onclick="showBottom(event)" class=" as-toolbar-tabs__item">BOTTOM</span>
-  </nav>
+  <div role="tablist" class="as-toolbar-tabs as-tabs as-tabs--xl" >
```
[the remark this answers is not in the document]
Comment (IagoLast) [reply examples/layouts/generic.html]: totally agreed, but according to [this](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html) the element with the role="tablist" should be a div

**Pull request merged**